quartz/util/theme.ts

```diff
@@ -26,7 +26,9 @@ const DEFAULT_SANS_SERIF =
 const DEFAULT_MONO = "ui-monospace, SFMono-Regular, SF Mono, Menlo, monospace"
 
 export function googleFontHref(theme: Theme) {
-  const { code, header, body } = theme.typography
+  const { code, header, body } = Object.fromEntries(
+    Object.entries(theme.typography).map(([key, value]) => [key, value.replace(/ /g, "+")]),
```

Comment:
actually fix a nasty bugs when the font has space in between, this URL is actually invalid apparently.

Reply:
wait really? all the default fonts in quartz has spaces and this hasn't been a problem

Reply:
hold on can you confirm this? which fonts does this break for?

Reply:
EB Garamond breaks

Reply:
and also all docs from google shows there needs a + between spaces https://developers.google.com/fonts/docs/css2#forming_api_urls

Reply:
oops didn't see this.

Reply:
Wait what the spaces works !!!

Ugh how did it fail before? LOL

we can just revert back to previous change